src/app/coursegrab/utils/constants.py

```diff
@@ -9,9 +9,12 @@
 
 # Possible values for notification
 ANDROID = "ANDROID"
-IPHONE = "IPHONE"
+IOS = "IOS"
 EMAIL = "EMAIL"
 NONE = "NONE"
 
+# Possible values for device type
+WEB = "WEB"  # + ANDROID, IOS
```

Comment:
is the comment for android/ios ppl logged in on web

Reply:
It means that possible values for device type is `WEB || ANDROID || IOS`